handler.go

```diff
@@ -375,6 +376,59 @@ func (fm *Frontman) sendResultsChanToHubWithInterval(resultsChan chan Result) er
 	}
 }
 
+// HealthCheck runs before any other check to ensure that the host itself and its network are healthly.
+// This is useful to confirm a stable internet connection to avoid false alerts due to network outages.
+func (fm *Frontman) HealthCheck() error {
+	hcfg := fm.Config.HealthChecks
+	if len(hcfg.ReferencePingHosts) == 0 {
+		fm.HealthCheckNotPassed = false
```

Comment:
`HealtchCheckPassed = true` would be nicer I think :)

Reply:
Okay, I will change names.